### PR TITLE
Enable code coverage report using coveralls.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

**Comment by bcrowe** [comment]
This has been on my mind! :clap:

**Comment by jadb** [comment]
:+1:  

One failed test though.

File: phpunit.xml.dist [hunk 1]
```diff
@@ -30,6 +31,20 @@
...
+        <blacklist>
+            <directory suffix=".php">./vendor/</directory>
+            <directory suffix=".ctp">./vendor/</directory>
+
+            <directory suffix=".php">./tests/</directory>
+            <directory suffix=".ctp">./tests/</directory>
+
+            <directory suffix=".php">./config/</directory>
+        </blacklist>
+    </filter>
```
Comment (markstory) [inline phpunit.xml.dist]: Indentation seems wrong.

**Comment by ADmad** [comment]
@markstory Made indentation consistent by changing to spaces.

@jadb The failed test is irrelevant. It's a dsn string parsing test which fail sporadically for some unknown reason.

**Comment by lorenzo** [comment]
nice :)

**Pull request merged**